### PR TITLE
Add axe selenium library

### PR DESCRIPTION
**Pull request by rkefi84**
We use the axe selenium library for our automated accessibility tests, and would be nice to have it included.

Here is a link for the library : https://github.com/mozilla-services/axe-selenium-python

Thanks!

### CODE REVIEW

**Comment by rkefi84** [comment]
Build Image step failing with this error :
```
ERROR: Could not build wheels for cryptography which use PEP 517 and cannot be installed directly
WARNING: You are using pip version 20.3.1; however, version 21.0.1 is available.
```
Getting the same error when i tried locally without the axe library too and got the same

File: Dockerfile [hunk 1]
```diff
@@ -93,6 +93,7 @@ RUN apk update \
     robotframework-requests==$REQUESTS_VERSION \
     robotframework-seleniumlibrary==$SELENIUM_LIBRARY_VERSION \
     robotframework-sshlibrary==$SSH_LIBRARY_VERSION \
+    axe-selenium-python \
```
Comment (UltimateDogg) [inline Dockerfile:96]: please tag to a version like the other libraries

**Review by UltimateDogg — CHANGES_REQUESTED** [review]

**Comment by UltimateDogg** [comment]
> 
> 
> Build Image step failing with this error :
> 
> ```
> ERROR: Could not build wheels for cryptography which use PEP 517 and cannot be installed directly
> WARNING: You are using pip version 20.3.1; however, version 21.0.1 is available.
> ```
> 
> Getting the same error when i tried locally without the axe library too and got the same

It seems there is some rust dependency we need now, that will be fixed in another pull

File: README.md [hunk 2]
```diff
@@ -26,6 +26,7 @@ The versions used are:
 * [Robot Framework Requests](https://github.com/bulkan/robotframework-requests) 0.8.0
 * [Robot Framework SeleniumLibrary](https://github.com/robotframework/SeleniumLibrary) 4.5.0
 * [Robot Framework SSHLibrary](https://github.com/robotframework/SSHLibrary) 3.5.1
+* [Axe Selenium Library](https://github.com/mozilla-services/axe-selenium-python) 2.6.1
```
Comment (ppodgorsek) [inline README.md:29]: The latest release is 2.1.6

File: Dockerfile [hunk 3]
```diff
@@ -100,6 +100,7 @@ RUN apk update \
...
```
Comment (ppodgorsek) [inline Dockerfile:103]:
```suggestion
    axe-selenium-python==$AXE_SELENIUM_LIBRARY_VERSION \
```

**Review by ppodgorsek — CHANGES_REQUESTED** [review]

**Review by ppodgorsek — APPROVED** [review]

**Pull request merged**